test/integration/overload_integration_test.cc

```diff
@@ -239,6 +253,36 @@ TEST_P(OverloadIntegrationTest, StopAcceptingConnectionsWhenOverloaded) {
   codec_client_->close();
 }
 
+TEST_P(OverloadIntegrationTest, BypassOverloadManagerTest) {
```

Comment:
I think you should test having a second listener that is not bypassed showing that that request gets throttled.

This would demonstrate only the non-bypassed listeners are impacted

Reply:
Added the tests.